_posts/2019-09-03-how-to-deploy-production-grade-kubernetes-cluster-aws.adoc

```diff
@@ -1851,15 +1908,21 @@ inputs = {
   # If your IAM users are defined in a separate AWS account (e.g., in a security account), pass in the ARN of an IAM
   # role in that account that ssh-grunt on the worker nodes can assume to look up IAM group membership and public SSH
   # keys
+  # TODO: It's not clear which account this is meant to be. I assume from the above that it's the security account.
+  # This is one of those places where we can assume they followed our other guides and just tell them what they should use here.
```

Comment:
Yes, security acct. It does say "e.g., in a security account" in the comment above.